src/core/Navigation/WizardNavigation/WizardNavigation/__snapshots__/WizardNavigation.test.tsx.snap

```diff
@@ -816,6 +816,7 @@ exports[`calling render with the same component on the same container does not r
     />
     <ul
       class="c3 fi-wizard-navigation_list"
+      role="list"
```

Comment:
As <ul> implicit ARIA role is list, adding it just in case seems a bit redundant. Or does it make some difference with assistive technologies?

Reply:
This is on purpose, just as a precaution due to this: https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns

Reply:
Ok, good to know!